runtime/controller.go

```diff
@@ -1268,6 +1274,18 @@ func (c *Controller) invoke(r *runtimev1.Resource) error {
 		c.Logger.Info("Reconciling resource", logArgs...)
 	}
 
+	// Start tracing span
+	tracerAttrs := []attribute.KeyValue{
+		attribute.String("name", n.Name),
+		attribute.String("kind", unqualifiedKind(n.Kind)),
+		attribute.Bool("deleted", inv.isDelete),
```

Comment:
Only add deleted if it's true

Reply:
Any particular reasons for that ? Won't it just add `deleted:false`